app/src/main/java/com/sun/moviedb_48/screen/adapter/MovieUpcomingAdapter.java

```diff
@@ -0,0 +1,87 @@
+package com.sun.moviedb_48.screen.adapter;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+import com.sun.moviedb_48.R;
+import com.sun.moviedb_48.data.model.Movie;
+import com.sun.moviedb_48.data.source.remote.OnFetchImageListener;
+import com.sun.moviedb_48.utils.Constant;
+import java.util.ArrayList;
+
+import static com.sun.moviedb_48.utils.Common.getImageUrl;
+
+public class MovieUpcomingAdapter extends RecyclerView.Adapter<MovieUpcomingAdapter.ViewHolder> {
+
+    static Context mContext;
```

Comment:
Không nên để static cho activity, view hay context để tránh memory leaks 
https://android.jlelse.eu/9-ways-to-avoid-memory-leaks-in-android-b6d81648e35e
bạn có thể lấy context từ itemView.getContext() và có thể truyền luôn Movie qua hàm bind của ViewHolder thay cho position 
```suggestion
    holder.bindView(mUpcommingMovies.get(position));
```